src/e2e/events/crud.spec.ts

```diff
@@ -0,0 +1,151 @@
+import { randomUUID } from "node:crypto";
+
+import { expect, test } from "../fixtures";
+
+const testData = {
+  name: "E2E Test Event",
+  location: "Playwright",
+  description: "This is a test event created by Playwright E2E tests.",
+  organizer: "KN Solvro",
+  slug: randomUUID(),
+};
+
+test.beforeEach(async ({ page }) => {
+  // Set the clock to a specific date and time
+  await page.clock.setFixedTime(new Date("2025-08-08T10:00:00"));
+  // Navigate to the dashboard
+  await page.goto("/dashboard");
+});
+
+test("event CRUD", async ({ page }) => {
+  await test.step("should create an event", async () => {
+    // Open up the event creation modal
+    await page.getByRole("button", { name: "Stwórz wydarzenie" }).click();
+    // Fill in the event details
+    await page.getByRole("textbox", { name: "Nazwa" }).fill(testData.name);
+    await page.getByRole("button", { name: "Data i godzina" }).click();
+    await page.getByRole("button", { name: "Wednesday, August 13th," }).click();
+    await page.locator('input[name="startTime"]').fill("13:30");
+    // NOTE: The date picker buttons don't have names, we should add them for easier testing
+    await page.getByRole("button", { name: "August 8th," }).first().click();
+    await page
+      .getByRole("button", { name: "Thursday, August 14th," })
+      .last()
+      .click();
+    await page.locator('input[name="endTime"]').fill("15:30");
+    await page
+      .getByRole("textbox", { name: "Miejsce (opcjonalnie)" })
+      .fill(testData.location);
+    await page
+      .getByRole("textbox", { name: "Opis" })
+      .fill(testData.description);
+    await page
+      .getByRole("textbox", { name: "Organizator (opcjonalnie)" })
+      .fill(testData.organizer);
+    // Skip the custom fields steps
+    await page.getByRole("button", { name: "Dalej" }).click();
+    await page.getByText("Spersonalizuj wydarzenie").waitFor();
+    // NOTE: find a way to have unique slugs for each test run
+    await page.getByRole("textbox", { name: "Slug" }).fill(testData.slug);
+    await page.getByRole("button", { name: "Dalej" }).click();
+    await page.getByText("Dodaj współorganizatorów").waitFor();
+    await page.getByRole("button", { name: "Dalej" }).click();
+    await page.getByText("Dodaj atrybuty").waitFor();
+    await page.getByRole("button", { name: "Utwórz wydarzenie" }).click();
+    // Wait for the event to be created
+    await page.waitForURL("/dashboard/events/*");
```

Comment:
nie jestem fanem testowania URL'i - imo to jest implementation detail, użytkownika raczej nie interesuje na jakim url'u się znajduję